assets/src/edit-story/components/keyboard/index.js

```diff
@@ -149,6 +149,31 @@ export function useGlobalKeyUpEffect(
   useKeyUpEffect(globalRef, keyNameOrSpec, callback, deps);
 }
 
+/**
+ * @param {string|Array|Object} keyNameOrSpec
+ * @param {Array|undefined} deps
+ * @return {boolean} Stateful boolean that tracks whether key is pressed.
+ */
+export function useGlobalIsKeyPressed(keyNameOrSpec, deps = undefined) {
```

Comment:
You should just use your own `useIsKeyPressed()` here. You will just need to do `if (!globalRef.current) {...}` that other methods are doing. And I'll also request you put that `if` into a separate private method, e.g. `getGlobalRef()` or such.

Reply:
Done.